src/App.jsx

```diff
@@ -5,46 +5,32 @@ import Courses from './components/Courses/Courses';
 import CreateCourse from './components/CreateCourse/CreateCourse';
 import Registration from './components/Registration/Registration';
 import Login from './components/Login/Login';
+import CourseInfo from './components/CourseInfo/CourseInfo';
 import './App.css';
-import { api_endpoints, storageKeys, paths } from './constants';
+import { API_ENDPOINTS, STORAGE_KEYS, PATHS } from './constants';
 
 const App = () => {
+  const env = 'http://localhost:4000';
   const [courses, setCourses] = useState([]);
   const [authors, setAuthors] = useState([]);
-  const [userName, setUserName] = useState(localStorage.getItem(storageKeys.userName) || '');
+  const [USER_NAME, setUserName] = useState(localStorage.getItem(STORAGE_KEYS.USER_NAME) || '');
```

Comment:
this one isn't a constant, I think you might have renamed it along with constant keys by accident